service/models/model_utils.py

```diff
@@ -28,3 +28,40 @@ def default(self, obj):
     if isinstance(obj, Availability):
       return obj.name
     return JSONEncoder.default(self, obj)
+
+from .wishlist import Wishlist
```

Comment:
Could you move this up to the top of the file? Linter wouldn't like this.

Reply:
This one is a little bit tricky. It has to be here because, if it is moved to the top, there will be a circular import.

Reply:
Ah damn. I tried to resolve it once, but got nowhere 😢  let's leave it then.